<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tuning</class>
 <widget class="QDialog" name="tuning">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>861</width>
    <height>711</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="But_Tune">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>310</y>
     <width>561</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color:transparent;
image: url(:/images/handpiece1.png);
border:none;}
QPushButton:focus {
    outline: none;
    border: none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="But_Next">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>570</y>
     <width>151</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;
image: url(:/images/nextbutton.png);
border:none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="But_Handpiece">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>370</y>
     <width>141</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;
border:none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="But_value">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>300</y>
     <width>131</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButRTune">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>620</y>
     <width>521</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(12, 40, 82);
color:rgb(255,255,255);
font:20pt;
font:bold;
</string>
   </property>
   <property name="text">
    <string>Ready For Tune</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblRTune">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>620</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/images/singletick.png);
background-color:transparent;
border:none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblTuned">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>670</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;
image: url(:/images/doubled.png);
border:none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="butTuned">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>670</y>
     <width>521</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(12, 40, 82);
color:rgb(255,255,255);
font:20pt;
font:bold;
</string>
   </property>
   <property name="text">
    <string> Tuned</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblinfo">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>110</y>
     <width>471</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(12, 40, 82);
color:rgb(255,255,255);
font:20pt;
font:bold;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>lblinfo</zorder>
  <zorder>But_Tune</zorder>
  <zorder>But_Next</zorder>
  <zorder>But_value</zorder>
  <zorder>But_Handpiece</zorder>
  <zorder>ButRTune</zorder>
  <zorder>lblRTune</zorder>
  <zorder>butTuned</zorder>
  <zorder>lblTuned</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
